specification/batch/resource-manager/Microsoft.Batch/stable/2021-06-01/BatchManagement.json

```diff
@@ -0,0 +1,5878 @@
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation was successful. The response contains the Batch service quotas for the subscription in the specified location.",
+            "schema": {
+              "$ref": "#/definitions/BatchLocationQuota"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Batch/locations/{locationName}/virtualMachineSkus": {
+      "get": {
+        "tags": [
+          "Location"
+        ],
+        "operationId": "Location_ListSupportedVirtualMachineSkus",
+        "x-ms-examples": {
+          "LocationListVirtualMachineSkus": {
+            "$ref": "./examples/LocationListVirtualMachineSkus.json"
+          }
+        },
+        "description": "Gets the list of Batch supported Virtual Machine VM sizes available at the given location.",
+        "parameters": [
+          {
+            "name": "locationName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The region for which to retrieve Batch service supported SKUs."
+          },
+          {
+            "name": "maxresults",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "format": "int32",
+            "description": "The maximum number of items to return in the response."
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "OData filter expression. Valid properties for filtering are \"familyName\"."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation was successful. The response contains the Batch service supported virtual machine vm sizes for the subscription in the specified location.",
+            "schema": {
+              "$ref": "#/definitions/SupportedSkusResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Batch/locations/{locationName}/cloudServiceSkus": {
```

Comment:
Any resource type MUST implement GET for collection and single resource regardless if it's a singleton or not.
Please add below API 
GET /subscriptions/{subscriptionId}/providers/Microsoft.Batch/locations/{locationName}/cloudServiceSkus/{skuName}

Refer 'Inconsistent or incomplete CRUD' in https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html